editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-conditionals.spec.browser2.tsx

```diff
@@ -62,6 +63,7 @@ async function createAndRenderProject() {
 }
 
 describe('a project with conditionals', () => {
+  setFeatureForBrowserTests('Conditional support', true)
```

Comment:
it'd be good to do a cleanup pass as well to reset the flag to the default value
```suggestion
  before(() => setFeatureEnabled('Conditional support', true))
  after(() => setFeatureEnabled('Conditional support', false))
```

Reply:
that's what setFeatureForBrowserTests does – check the implementation! :)

Reply:
oooh sorry, I should have checked better. ignore me!